src/test_workflow/benchmark_test/benchmark_test_cluster.py

```diff
@@ -143,12 +141,12 @@ def setup_cdk_params(self, config: dict) -> dict:
             artifact_url = self.manifest.build.location if isinstance(self.manifest, BundleManifest) else \
                 f"https://artifacts.opensearch.org/snapshots/core/opensearch/{self.manifest.build.version}/opensearch-min-" \
                 f"{self.manifest.build.version}-linux-{self.manifest.build.architecture}-latest.tar.gz"
-            if not self.args.insecure and semver.compare(self.manifest.build.version, '2.12.0') != -1:
-                need_strong_password = True
+            if not self.args.insecure:
+                password = 'myStrongPassword123!' if get_password(self.manifest.build.version) == 'myStrongPassword123!' else None
         else:
```

Comment:
`password = get_password(self.manifest.build.version)` having this should solve the purpose right @derek-ho ?
Adding @peterzhuamazon

Reply:
Agreed with @prudhvigodithi, we can just assign the value of get_password to password.

Reply:
I was also confused about this, but it seems like this specific test is trying to set "myStrongPassword123!" or None depending on the version, instead of "myStrongPassword123!" or "admin". Thus, I had to make the usage of the util function in this way.

Reply:
Hey! so the clusters are deployed with and without security for performance runs. Thats why password is not always required. If required, pass the right one else `None`